lib/spack/llnl/util/filesystem.py

```diff
@@ -2458,6 +2461,26 @@ def establish_link(self):
                 self._link(library, lib_dir)
 
 
+def make_package_test_rpath(pkg, test_dir):
+    """Establishes a temp rpath for the pkg in the testing directory
+    so an executable can test the libraries/executables with proper access
+    to dependent dlls
```

Comment:
It would be useful if the method name or docstring "opener" mention this is for Windows.